src/main/java/io/vertx/core/impl/launcher/commands/BareCommand.java

```diff
@@ -107,6 +113,25 @@ public void setClusterHost(String host) {
     this.clusterHost = host;
   }
 
+  /**
+   * The Vert.x options, it can be a json file or a json string.
+   *
+   * @param vertxOptions the configuration
+   */
+  @Option(longName = "vertx-options", argName = "options")
```

Comment:
just name it _options_ there isn't a need for two different names

Reply:
Agreed. `options` is fine.

Reply:
Done